.github/workflows/call-pr-1-ci.yml

```
@@ -0,0 +1,26 @@
+# This workflow is used to convert the `.version` in the `metadata.yaml` file into a valid `git tag` on push to `main`.
+# We use the `.version` field in that file to denote the version of the config once a PR is merged.
+name: pr-1-ci.yml
+on:
+  pull_request:
+    branches:
+      - 'release-*'
+      - 'dev-*'
+    paths-ignore:
+      # These are ignored because they don't have anything to do with the model itself
+      - .github/**
+      - tools/**
+      - doc/**
+      - .*
+      - metadata.yaml
+      - README.md
+jobs:
+  call:
+    # We simply call the workflow on main so we don't have to propagate CI changes to
+    # multiple config branches all the time.
+    uses: codegat-test-org/configs/.github/workflows/pr-1-ci.yml@main
```

```suggestion
    uses: access-nri/access-om2-configs/.github/workflows/pr-1-ci.yml@main
```

Still pointing to your test org. Please check my suggestion is correct.